### DISCUSSION TITLE
Advice: 2 TRV (climate) in the same room, prioritize one radiator, control individual when curtain is closed

### DISCUSSION THREAD

**Discussion by Eagleman7** [jmcollin78/versatile_thermostat#387] [General]
First off thanks for this amazing integration!

I'm using Versatile Thermostat in all my rooms. My living room is a bit special, it's a room combined with the kitchen which also has a 
radiator. So 2 radiator in one big combined room. 

The radiator in the kitchen zone is "cheaper" to turn on than the one in the living room. I would prefer this one to start at 100% and if it's not enough then turn on the radiator in the living room as well.

Another use case I'm trying to build is to turn off the Versatile Thermostat climate.living_room (which has both living room and kitchen radiators assigned) when the curtains in front of the living room radiator. I have another Versatile Thermostat `climate.kitchen` integration with only the kitchen radiator assigned. However when turning on `climate.kitchen` it also enables `climate.living_room`.

I would like to know if it's possible what I'm trying to do with Versatile Thermostat and advice on how to do it.

Thanks!

**Comment by jmcollin78** [comment]
Hello @Eagleman7 ,

I suppose that you have `over_switch` VTherm.

Out of the box, it is not possible to do what you want to do. The only solution I see it to build a fake switch which will turn on / off with your proper logic the heaters depending on the power_percent (or on on_percent) attributes of the VTherm.

The logic implemented in the virtual swicth could be something like:
- if on_percent < 0.5 only turn on one heater,
- if on_percent >= 0.5 and < 1, turn on the first, and turn on the second for a few minutes (depending on the on_percent),
- if on_percent >= 1, turn on the both heater.

This could make the trick but it not so easy to do. You can be inspired by what is done for Heatzy in the README.
